build.py

```diff
@@ -122,9 +124,9 @@ def md5(fname):
 config['libdevice_versions'] = [config['version'].split(".")[0]]
 
 config['linux'] = {
-    'blob': 'cuda_11.8.0_520.61.05_linux.run',
-    'ppc64le_blob': 'cuda_11.8.0_520.61.05_linux_ppc64le.run',
-    'aarch64_blob': 'cuda_11.8.0_520.61.05_linux_sbsa.run',
+    'blob': 'cuda_12.3.0_545.23.06_linux.run',
+    'ppc64le_blob': 'cuda_12.3.0_545.23.06_linux_ppc64le.run',
+    'aarch64_blob': 'cuda_12.3.0_545.23.06_linux_sbsa.run',
```

Comment:
Looks like you could extract everything that needs to be changed between versions into a local conda_build_config.yaml